webapp/src/components/expanded_view/recording_info_prompt.tsx

```diff
@@ -81,7 +88,7 @@ export default function RecordingInfoPrompt(props: Props) {
                     defaultMessage: 'Your recording will end in {count, plural, =1 {# minute} other {# minutes}}.'}
                 , {count: getMinutesLeftBeforeEnd()})}
                 confirmText={formatMessage({defaultMessage: 'Dismiss'})}
-                onClose={() => updateDismissedAt(Date.now())}
+                onClose={updateDismissed}
```

Comment:
Not strictly necessary, could use the original, but thought it's good to be consistent in case we add something in the future to the `updateDismissed` block that affects here.